docs/5-design-patterns/hometask/shipper.ts

```diff
@@ -0,0 +1,59 @@
+import { ShippingCompany, TypeOfPackage } from "./enum";
+
+interface IShipper {
+  rate: number;
+  getCost(wieght: number): number;
+}
+
+class AirEastShipper implements IShipper {
```

Comment:
Optional: The implementation looks good, but you can write each implementation in separate files.